Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -64,18 +71,28 @@ void IChangeTargetRenderSettings( RenderSettingsData renderSettings )
 
 	void ApplyRenderSettings()
 	{
-		Color skyboxColor = Color.Lerp( _currentRenderSettings.dayColor,
-		                                _currentRenderSettings.nightColor,
+		Color skyboxColor = Color.Lerp( _currentRenderSettings.daySettings.color,
+		                                _currentRenderSettings.nightSettings.color,
 		                                timeOfDay );
 		RenderSettings.skybox.SetColor( "_Tint", skyboxColor );
 
-		RenderSettings.fogColor = _currentRenderSettings.fogSettings.color;
-		RenderSettings.fogDensity = _currentRenderSettings.fogSettings.density;
+		RenderSettings.fogColor = Color.Lerp( _currentRenderSettings.daySettings.fogColor,
```

Comment:
I'm not seeing where this "RenderSettings" comes from. Is it a class with static fields?